### DISCUSSION TITLE
How should I do to prohibit a software to follow Auto Dark Mode to enable dark mode?

### DISCUSSION THREAD

**Discussion by lovimnnqbi** [AutoDarkMode/Windows-Auto-Night-Mode#856] [Q&A]
各位大佬，我想要禁止某一个软件跟随Auto Dark Mode启用深色模式应该怎样做？比如小狼毫输入法，当Auto Dark Mode启用深色模式时，小狼毫输入法打字时界面不能正常显示。小狼毫输入法的版本为v0.16.1。谢谢！
--------
How should I do to prohibit a software to follow Auto Dark Mode to enable dark mode? For example, the RIME, when Auto Dark Mode enabled dark mode, the RIME's typing interface can not show normally. The version of the RIME is v0.16.1. Thank you!
--------
Light Mode:
![image](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/124759361/fe6c37e6-2142-49db-ae0c-cdc0e0c4955f)
Dark Mode:
![image](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/124759361/ad2e08c0-400c-4d1c-be2f-504ae3239eaa)

**Comment by Armin2208** [comment]
This is an issue of the app itself. You should contact the developer to fix auto theme switching or adding an option to deactivate auto theme listener. Auto Dark Mode can't influence third party apps.

**Reply by lovimnnqbi** [reply]
OK, I got it, Thank you!
